### ISSUE TITLE
Release for GHC 9.4.1

### ISSUE THREAD

**Issue by bgamari** [haskell/containers#845]
We are coming up on the final release for 9.4.1 and at this point it appears no changes will be necessary in `containers`. It would be appreciated if a prompt release could be made.

**Comment by treeowl** [comment]
Thanks for the heads up. I'll do it tomorrow.

**Comment by bgamari** [comment]
How is this going, @treeowl ?

**Comment by treeowl** [comment]
'Twill be done today

**Comment by bgamari** [comment]
Any progress on this, @treeowl ?

**Comment by treeowl** [comment]
Yes. I am making progress on this. I'm sorry it wasn't done yesterday, but it will definitely be done today.
